cmd/bundle/run.go

```diff
@@ -34,6 +34,12 @@ func promptRunArgument(ctx context.Context, b *bundle.Bundle) (string, error) {
 		inv[title] = k
 	}
 
+	// Include scripts in the prompt options
+	for k := range b.Config.Scripts {
+		title := "Script: " + k
+		inv[title] = k
+	}
```

Comment:
This means it's not autocompletable.

Can this be integrated as a first-class runnable in `bundle/run`?

Then we get both autocompletion and disambiguation.

Reply:
I can look into that as a followup. I'll take a note.